molecule/driver/dockerdriver.py

```diff
@@ -22,7 +22,10 @@
 import io
 import json
 
-import docker
+try:
+    import docker
+except ImportError:
+    LOG.error('Driver missing, install docker-py!')
```

Comment:
LOG hasn't been imported at this point so this fails. In hindsight I think what we are really going to want is something like this.

```
try:
    import docker
except ImportError:
    sys.exit('ERROR: Driver missing, install python-docker!')
```

This way we don't require the LOG import and the program exits with a non-zero status.